ops/ccs-ops-misc/load_test/common/bene_tests.py

```diff
@@ -0,0 +1,178 @@
+'''Locust tests that require a pool of beneficiary IDs.'''
+
+import random
+
+from common.bfd_user_base import BFDUserBase
+from common.url_path import create_url_path
+from common import data, db
+
+
+class BeneTestUser(BFDUserBase):
+    '''Locust tests that require a pool of beneficiary IDs.'''
+
+    # Mark this class as abstract so Locust knows it doesn't contain Tasks
+    abstract = True
+
+    # Should we use the Table Sample feature of Postgres to query only against a portion of the
+    # table?
+    USE_TABLE_SANPLE = True
```

Comment:
While consistent, I'm only curious. Is _sanple_ intentional throughout this changeset?

Reply:
Ugh... I guess I just cam't spell.